Source/CombatExtended/CombatExtended/WorldObjects/WorldObjectDamageWorker.cs

```diff
@@ -37,12 +37,13 @@ public virtual float CalculateDamage(ThingDef projectile, Faction faction)
             var result = FragmentsPotentialDamage(projectile) + ExplosionPotentialDamage(projectile) + FirePotentialDamage(projectile) + EMPPotentialDamage(projectile, empModifier);
             //Damage calculated as in-map damage, needs to be converted into world object damage. 3500f experimentally obtained
             result /= 3500f;
```

Comment:
Are we actually dividing the xml-defined damage by the factor of 3500? Or is it only being applied to the damage being automatically calculated from the shell properties?